libvast/src/plugin.cpp

```diff
@@ -380,6 +381,18 @@ store_plugin::make_store(system::accountant_actor accountant,
                                                  std::move(path), name());
 }
 
+// -- pipeline_operator_plugin -------------------------------------------------
+
+std::pair<std::string_view, caf::expected<std::unique_ptr<pipeline_operator>>>
+pipeline_operator_plugin::make_pipeline_operator(
+  std::string_view pipeline) const {
+  // FIXME: Remove default impl.
```

Comment:
Can we have a story for this? Might forget about it in the future

Reply:
I can deliver a commit instead: https://github.com/tenzir/vast/pull/2877/commits/13c898d01662ed17f97d18770af0d63f3edbd687

Reply:
Ah, i thought we are not supporting all the pipeline_operator_plugin yet. I was looking at the operators directory, but these were only aggregation function, not a pipeline_operator_plugins :D. Thanks for that